vendor/github.com/weppos/publicsuffix-go/publicsuffix/rules.go

```diff
@@ -0,0 +1,8786 @@
+// This file is automatically generated
+// Run "go run cmd/gen/gen.go" to update the list.
+
+package publicsuffix
+
+const defaultListVersion = "PSL version 6f03f4 (Fri May 31 16:38:49 2019)"
```

Comment:
How often is this updated and what are the consequences of being out of date? Do we fail open or closed on unknown URL suffixes?